packages/FrontC/FrontC.4.1.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Parses C programs to an abstract syntax tree"
+description:
+  "FrontC provides a C parser and an OCaml definition of an abstract syntax treee for the C language. It also includes AST pretty-printers in plain and XML formats."
+maintainer: ["Ivan Gotovchits <ivg@ieee.org>"]
+authors: ["Hugues Cassé <casse@irit.fr> et al"]
+license: "LGPL-2.0-only"
+tags: ["FrontC" "C" "parser" "XML"]
+homepage: "https://github.com/BinaryAnalysisPlatform/FrontC"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/FrontC/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.7"}
+  "menhir"
```

Comment:
```suggestion
  "menhir" {>= "20180523"}
```
since you are using `menhir --infer-write-query`

Reply:
Definitely! Thanks :)